src/components/Tabs.js

```diff
@@ -59,11 +61,22 @@ class SimpleTabs extends React.Component {
     const { classes } = this.props;
     const { value } = this.state;
     const jobColumns = ["Name", "Description", "Created At"];
-    const datasetColumns = ["URN", "Created At"];
+    const datasetColumns = ["Name", "urn", "Created At"];
```

Comment:
We'll want to also add a column for dataset descriptions.

Reply:
Cool, opened #5 for that